storage/bootstrap.go

```diff
@@ -194,6 +160,46 @@ func (m *bootstrapManager) Bootstrap() error {
 	return multiErr.FinalError()
 }
 
+func (m *bootstrapManager) Report() {
+	if m.IsBootstrapped() {
+		m.status.Update(1)
+	} else {
+		m.status.Update(0)
+	}
+
+}
+
+func (m *bootstrapManager) targetRanges(at time.Time) []bootstrap.TargetRange {
```

Comment:
It's probably fine, I'd rather have all public APIs at the top of the function and the private implementation details at the bottom. If we really need to git blame this function, we can still do it with some extra effort.